pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -791,9 +791,26 @@ public void start() throws PulsarServerException {
             } catch (KeeperException.NodeExistsException e) {
                 long ownerZkSessionId = getBrokerZnodeOwner();
                 if (ownerZkSessionId != 0 && ownerZkSessionId != zkClient.getSessionId()) {
-                    log.error("Broker znode - [{}] is own by different zookeeper-ssession {} ", brokerZnodePath,
-                            ownerZkSessionId);
-                    throw new PulsarServerException("Broker-znode owned by different zk-session " + ownerZkSessionId);
+                    if (conf.isRunningStandalone()) {
```

Comment:
I don't feel it is a right place.. this is something related to only standalone server and I think it makes more sense to add startup-clean up at `PulsarStandaloneStarter` because we can have different `LoadManager` implementation and we don't want to repeat this logic which is specific to standalone broker.

Reply:
Sure, we can create a no-op load manager and use that in standalone. I'll fix this